Sources/_SwiftSupport.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+#if swift(>=4.1)
```

Comment:
I misread that at first and thought you made a typo, until I saw the `else` clause.
Maybe make this `#if !swift(>=4.1)`?